OS X-related check implemented in bash - not needed on Linux, just remove it
--- ilmbase-1.0.2/configure.ac.orig	2010-07-21 02:52:20.000000000 +0200
+++ ilmbase-1.0.2/configure.ac	2010-10-12 20:30:30.043738529 +0200
@@ -110,16 +110,6 @@
      osx_arch_flags="-arch ${osx_arch// / -arch }"
      CXXFLAGS="$CXXFLAGS $osx_arch_flags"
 
-     dnl If multiple archs specified, make sure that --disable-dependency-tracking was also specified
-     if test "$enable_dependency_tracking" != no ; then
-        osx_arch_array=( ${osx_arch} )
-        if test ${#osx_arch_array[@]} -gt 1 ; then
-          AC_MSG_ERROR([building multiple OS X architectures requires --disable-dependency-tracking.
-Please re-run configure with these options:
-  --enable-osx-arch="${osx_arch}" --disable-dependency-tracking
-  		  ])
-        fi
-     fi
   fi
 
   dnl OSX SDK specification
